README.md

```diff
@@ -37,59 +37,101 @@ To use the SDK, you will need Cisco Webex credentials. If you do not already hav
 
 See [the detailed docs](https://webex.github.io/spark-js-sdk/) for more usage examples.
 
-### Node.js
-
 You will need to set the following environment variable:
+
 - `CISCOSPARK_ACCESS_TOKEN`
 
 ```javascript
-const assert = require(`assert`);
-assert(process.env.CISCOSPARK_ACCESS_TOKEN, 'This example assumes you have set your access token as an environment variable');
 const ciscospark = require(`ciscospark`);
-ciscospark.rooms.create({title: `My First Room`})
-  .then((room) => {
-    return Promise.all([
-      ciscospark.memberships.create({
-        roomId: room.id,
-        personEmail: `alice@example.com`
-      }),
-      ciscospark.memberships.create({
-        roomId: room.id,
-        personEmail: `bob@example.com`
-      }),
-    ])
-      .then(() => ciscospark.messages.create({
-        markdown: `**Hi Everyone**`,
-        roomId: room.id
-      }));
+const spark = ciscospark.init({
+  credentials: {
+    access_token: process.env.CISCOSPARK_ACCESS_TOKEN
```

Comment:
Maybe we can put a comment here that the access token doesn't have to come from an env var with this setup.